snapcraft/internal/errors.py

```diff
@@ -202,7 +202,7 @@ def __init__(self, remote):
         super().__init__(remote=remote)
 
 
-class InvalidDesktopFileError(SnapcraftError):
+class InexistantFile(SnapcraftError):
```

Comment:
How about `MissingDesktopFileError`. Also, it doesn't need to accept a message anymore since it's suitably specific-- just put the message here in the exception.